modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavascriptClientCodegen.java

```diff
@@ -500,7 +509,7 @@ public String toVarName(String name) {
 
         // camelize (lower first character) the variable name
         // pet_id => petId
-        name = camelize(name, true);
+        //name = camelize(name, true);
```

Comment:
@CodeNinjai Commenting out `camelize` will probably impact JS ES5 output as well. Shall we keep it as it's?

cc @frol @cliffano

Reply:
@CodeNinjai Given the code style guides for JS, camel-case should be used for variable and method names. Please, bring it back.

Also, it seems that the custom `name.matches(...)` should be simply placed into `sanitizeName` once instead of copy-pasting.